lib/Steps/PromptForParameters.ts

```diff
@@ -120,7 +120,16 @@ export class PromptForParameters extends BaseStep {
   }
 
   private _validateAuthToken(input: string): boolean | string {
-    if (!input.match(/[0-9a-f]{64}/g)) {
+    const isOrgToken = input.startsWith('sntrys_');
+
+    if (isOrgToken) {
+      if (!isValidOrgToken(input)) {
+        return 'Make sure you copied the correct auth token, it should start with sntrys_ and consist of >140 base64 characters';
```

Comment:
I would maybe just phrase it like this

```suggestion
        return 'Make sure you correctly copied your auth token.';
```

The additional info is not gonna help people I believe unless there is an exact number of characters we can give them.